.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Remove this file. Rest looks good to me. 
@smaranjitghose any, reviews?

Reply:
Yep. It's cool!